values.yaml

```diff
@@ -0,0 +1,89 @@
+# Default values for nbgallery.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: nbgallery/nbgallery
+  tag: latest
+  pullPolicy: Always
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+email:
+  domain:
+  address:
+  server:
+  port:
+mysql:
+  host: nbgallery-mysql
+  port: "3306"
+  user: nbgallery
+  existingSecret: mysql-password
+  db: gallery
+solr:
+  host: nbgallery-solr-svc
+  port: 8988
+persistence:
+  annotations: {}
+  accessMode: ReadWriteOnce
+  size: 10Gi
+  storageClass: default
```

Comment:
"default" is actually a bad default.  I think this just has to be left out to pick up the default of the provider (gp2 for AWS for example)

Reply:
In any kubernetes cluster default can be used. For example, azure's default is azure-disk, but you can use it by just saying `default`

Reply:
And this is in the values.yaml file, so the user can easily change the storage class to something they want.

Reply:
IN my testing case, on AWS, EKS, "default" here actually prevented it from picking the cluster default of 'gp2' and thus failed to allocate a volume, which then took a bit of time to figure out what was going on.

Reply:
So...had another thought that I might explore which is building a docker container from the official solr docker container, that has the sunspot stuff baked in so it just works.

Reply:
Hmm alright then. Maybe kubernetes isn't as smart as I thought

Reply:
would leaving it blank make it take the default?